docs/source/kale.utils.rst

```diff
@@ -14,6 +14,14 @@ kale.utils.csv\_logger module
    :undoc-members:
    :show-inheritance:
 
+kale.utils.download module
```

Comment:
Why duplicate these `download` updates in two PRs and requested two reviews? This is inefficient and against our 3R principle. Try to think about the efficiency of your workflow before commits.